test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1078,6 +1078,8 @@ UCS_TEST_SKIP_COND_P(test_ucp_am_nbx_send_copy_header, all_protos,
     size_t header_length;
     size_t data_length;
 
+    UCS_TEST_SKIP_R("Skip while fixing CI hang bug.");
```

Comment:
`FIXME: Disabled due to unresolved failure`

Reply:
`FIXME` is a convention we use in this situations?

Reply:
Kind of (search in repo).